File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--add-windows-forms-controls-to-office-documents_2.vb
        Dim salesButton As Microsoft.Office.Tools.Word.Controls.Button

        Dim extendedDocument As Document = Globals.Factory.GetVstoObject(Me.Application.ActiveDocument)

        extendedDocument.Paragraphs(1).Range.InsertParagraphBefore()
        salesButton = extendedDocument.Controls.AddButton( _
            extendedDocument.Paragraphs(1).Range, 75, 18, "salesButton")
        salesButton.Text = "Calculate Sales"

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--add-windows-forms-controls-to-office-documents_2.cs
            Microsoft.Office.Tools.Word.Controls.Button salesButton;

            Document extendedDocument = Globals.Factory.GetVstoObject(this.Application.ActiveDocument);


            extendedDocument.Paragraphs[1].Range.InsertParagraphBefore();
            salesButton = extendedDocument.Controls.AddButton(
                extendedDocument.Paragraphs[1].Range, 75, 18, "salesButton");
            salesButton.Text = "Calculate Sales";